.github/workflows/deploy-gh-pages.yml

```
@@ -0,0 +1,51 @@
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - uses: "actions/checkout@v4"
+
+      - uses: "actions/setup-node@v4"
+        with:
+          node-version: "22"
+
+      - name: "Get npm cache directory"
+        id: "get-npm-cache-dir"
+        run: "echo 'dir=$(npm config get cache)' >> ${GITHUB_OUTPUT}"
```

actionlint

shellcheck reported issue in this script: SC2016:info:1:6: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)